### ISSUE TITLE
Update to MC 1.21(.1)

### ISSUE THREAD

**Issue by Kepler-17c** [Natanaelel/enderman_tweaks#3]
Since the latest version doesn't mention MC 1.21(.1), this issue doubles as a question and a request:

Is ET 0.3.0 compatible with MC 1.21(.1)? It should be marked for Modrinth if that's the case. Currently, it displays MC 1.20.1 as the only supported version.

Otherwise, please update this mod for MC 1.21(.1). I love that it prevents real mob griefing only rather than turning off a ton of game mechanics like the game rule does.

Thanks in advance.

**Comment by Natanaelel** [comment]
No, it is not compatible with 1.21. Yes I will update to 1.21 real soon.

**Comment by Kepler-17c** [comment]
Thank you for the quick response ^^

Looking forward to it.
